expression/bench_test.go

```diff
@@ -255,7 +332,7 @@ func genVecExprBenchCase(ctx sessionctx.Context, testCase vecExprBenchCase) (Exp
 	return expr, input, output
 }
 
-func TestVectorizedExpression(t *testing.T) {
+func TestVectorizedEvalOneVec(t *testing.T) {
 	ctx := mock.NewContext()
 	for _, testCase := range vecExprBenchCases {
```

Comment:
After this PR, contributors who reimplemented a new scalar function should update `vecExprBenchCases` to get that function benchmarked?

Reply:
Yes. 
`TestVectorizedBuiltinFunc` will read this list to do some correctness tests by comparing the row-based results and the vectorized results. 
`BenchmarkVectorizedBuiltinFunc` will use it to do some benchmarks.
And `TestVectorizedEvalOneVec` is the individual test for `evalOneColumn` and `evalOneVec`, which are used in projection operators.

Reply:
If each PR needs to modify this list, I think the merge conflict between PRs would be very heavy.

Reply:
But these conflicts are small(one line).
Would you have any other suggestions?

Reply:
The `vecExprBenchCases` is split to multiple lists by function name.
PTAL.